server/routerlicious/packages/routerlicious-base/package.json

```diff
@@ -80,7 +80,6 @@
     "nconf": "^0.12.0",
```

Comment:
I thought it was necessary to keep it because in app.ts I used it to define the `tokens` type in the `computeExtraProperties` function:
`const computeExtraProperties = (tokens: morgan.TokenIndexer, req: express.Request, res: express.Response) => {`

Is there a way to work around that?

Reply:
Just saw your other comment, which precisely answers my question :) Will send a new iteration soon